frontend/src/views/Notifications/NotificationMenu/NotificationMenu.jsx

```diff
@@ -0,0 +1,76 @@
+import BCBox from '@/components/BCBox'
+import {
+  NOTIFICATIONS,
+  NOTIFICATIONS_SETTINGS
+} from '@/constants/routes/routes'
+import breakpoints from '@/themes/base/breakpoints'
+import { NotificationTabPanel } from './components/NotificationTabPanel'
+import { AppBar, Tab, Tabs } from '@mui/material'
+import { PropTypes } from 'prop-types'
+import { useEffect, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { Notifications, NotificationSettings } from '.'
+
+function a11yProps(index) {
+  return {
+    id: `full-width-tab-${index}`,
+    'aria-controls': `full-width-notifications-tabs-${index}`
+  }
+}
+
+export function NotificationMenu({ tabIndex }) {
```

Comment:
is it possible to increase the size of email & notification slightly, as it seems smaller than the checkbox here.
<img width="102" alt="image" src="https://github.com/user-attachments/assets/eca113c4-ca69-47f0-bab5-fb19921514e6">

Reply:
I increased the icon size and aligned them LTR as suggested. It looks much better now!